docs/usage.md

```diff
@@ -44,87 +63,196 @@ TREATMENT_REP3,AEG588A6_S6_L003_R1_001.fastq.gz,
-| `sample`       | Custom sample name. This entry will be identical for multiple sequencing libraries/runs from the same sample. Spaces in sample names are automatically converted to underscores (`_`). |
-| `fastq_1`      | Full path to FastQ file for Illumina short reads 1. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
-| `fastq_2`      | Full path to FastQ file for Illumina short reads 2. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
+| Column    | Description                                                                                                                                                                            |
+| --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `sample`  | Custom sample name. This entry will be identical for multiple sequencing libraries/runs from the same sample. Spaces in sample names are automatically converted to underscores (`_`). |
+| `fastq_1` | Full path to FastQ file for Illumina short reads 1. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
+| `fastq_2` | Full path to FastQ file for Illumina short reads 2. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
 
 An [example samplesheet](../assets/samplesheet.csv) has been provided with the pipeline.
 
+### Genomes input
+
+Magmap needs two mandatory input to run. One, the samplesheet, was explained above while the second, genome input needs to be specified with the option `--genomeinfo`. The file is a `.csv` file and it requires three columns: accno, genome_fna, genome_gff. it looks as follow:
+
+```csv title="samplesheet.csv"
+accno,genome_fna,genome_gff
+GCA_002688505,./genomes/GCA_002688505.fna,./genomes/GCA_002688505.gff
+GCA_002688515,./genomes/GCA_002688515.fna,
+```
+
+N.B.: you don't need to provide gff, Prokka can handle if you only provide fasta files but we recommend when adding gff that the file is generated by using prokka in order to don't get any conflict with other generated gffs.
+
+### Other inputs (optional)
+
+Magmap can handle several types of input that can be used for different purpose.
+
+#### Indexes input
+
+The indexes input is used by Sourmash to select genomes that can downloaded in a second step and added to the pipeline.
+It is provided with the `--indexes` parameter and it is a path (local or remote).
+
+```
+https://github.com/nf-core/test-datasets/raw/magmap/testdata/sourmash_test.index.sbt.zip
+```
```

Comment:
What is this? Is this meant t obe an example? If so add the parameter before the URL

Reply:
And don't forget to specify the code block langauge/syntax